spk/cops/src/wizard_templates/install_uifile.yml

```diff
@@ -0,0 +1,13 @@
+COPS_CONFIGURATION_FIRST_STEP_TITLE: "COPS configuration"
+
+EXISTING_CALIBRE_DIRECTORY_DESCRIPTION: "Existing Calibre library directory. WARNING: has to end with a / and never use your web directory"
+EXISTING_CALIBRE_DIRECTORY_LABEL: "Calibre directory"
+COPS_CATALOG_TITLE_DESCRIPTION: "Catalog title."
+COPS_CATALOG_TITLE_LABEL: "Catalog title"
+
+COPS_CONFIGURATION_SECOND_STEP_TITLE: "COPS configuration (next)"
+DO_YOU_WANT_TO_USE_COPS_WITH_A_KOBO_DESCRIPTION: "Do you want to use COPS with a Kobo eReader?"
+DO_YOU_WANT_TO_USE_COPS_WITH_A_KOBO_LABEL: "Kobo"
+
+DSM_PERMISSIONS_TITLE: "Attention! DSM Permissions"
+DSM_PERMISSIONS_TEXT: "Permissions are managed with the group <b>http</b> in DSM.<br>The group 'users' is no longer used as of DSM 6.<br>Package user (= sc-cops) will not appear on most UI settings.<br>Including the following:<br>- Application privilege's permission viewer<br>- FPT's chroot user selector<br>- File Stations's<br>- Change owner<br>- Shared Links Manager -> Enable secure sharing<br><br>The only exceptions are:<br>- Control Panel > Shared Folder > Edit > Permission > System internal user<br>- ACL editor<br>"
```

Comment:
Please provide a link to the permission management wiki page. This is mandatory and avoids discussions as in #5832.

The text _The group 'users' is no longer used as of DSM 6._ and more is related to the DSM5 to DSM 6 migration and is obsolete.

Reply:
Indeed. Fixed.